modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -106,6 +110,14 @@ The following snippet sets DNS configuration on the host.
         - 8.8.8.8
 ...
 ----
+<1> You must configure an interface when configuring the DNS resolver. A manifest that changes the DNS resolver configuration and does not include an interface configuration will not work.
+
+<2> You must set the `auto-dns` configuration setting on the interface configuration to `false` if using DHCP on the interface, or if you configure a static IP address and a default gateway on the interface.
+
+[IMPORTANT]
+====
+You cannot use `br-ex`, an Open vSwitch brige, as the interface when configuring DNS resolvers.
```

Comment:
*bridge

I might also reword this as "You cannot use `br-ex`, an OVNKubernetes-managed Open vSwitch bridge" to make it clear that it's not all OVS bridges that are a problem.